sos/report/plugins/apache.py

```diff
@@ -130,8 +130,39 @@ def setup(self):
                 for log in logs:
                     self.add_copy_spec(f"{ldir}/{log}")
 
+        self.add_copy_spec([
+            "/var/lib/httpd/*"
```

Comment:
minor nit, add comma at the end, if there are new changes afterwards, it would be a one-line change rather than 2